### PR TITLE
Bump smart_proxy_dns_powerdns to 0.2.1 (RPM)

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ekohl** [comment]
For EL7 I opened https://bugzilla.redhat.com/show_bug.cgi?id=1310064 to get rubygem-pg packaged.

**Comment by domcleal** [comment]
I packaged it into epel-testing, it'll appear there in a day or two.

If it works, please add karma to https://bodhi.fedoraproject.org/updates/rubygem-pg-0.18.4-2.el7 for it to be accepted into the distribution.

I don't think our repoclosure tests enable epel-testing, so we might have to wait for it to reach stable or enable testing.

**Comment by ekohl** [comment]
Thanks. I'm in favour of it reaching stable so I'll test it when it's built.

**Comment by domcleal** [comment]
I'm now able to push the EPEL package to stable, but I wondered if you'd had chance to test it yet?

**Comment by ekohl** [comment]
I think I did install and test it, but I double check when I'm at home.

**Comment by mmoll** [comment]
what's the status here?

File: rubygem-smart_proxy_dns_powerdns/rubygem-smart_proxy_dns_powerdns.spec [hunk 1]
```diff
@@ -78,6 +79,9 @@ cp -pa .%{gem_instdir}/config/dns_powerdns.yml %{buildroot}%{foreman_proxy_setti
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Thu Apr 28 2016 Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl> 0.2.1-1
+- Bump to version 0.2.1
```
Comment (mmoll) [inline rubygem-smart_proxy_dns_powerdns/rubygem-smart_proxy_dns_powerdns.spec]: this is autogenerated when tagged, so this must get deleted here

**Comment by ekohl** [comment]
Updated, only blocked on rubygem-pg making it to EPEL7 now.

**Comment by domcleal** [comment]
It'll be another couple of weeks or one additional karma point before the package is accepted to EPEL7.

**Comment by domcleal** [comment]
Merged and will retag to 1.12, thanks @ekohl.

**Pull request merged**